templates/agent/src/handlers/game.ts

```
@@ -0,0 +1,57 @@
+  // URLs for each game type
+  const gameUrls: { [key: string]: string } = {
+    wordle: "https://framedl.xyz",
+    slot: "https://slot-machine-frame.vercel.app",
+  };
+  // Respond with the appropriate game URL or an error message
+  switch (params.game) {
+    case "wordle":
+    case "slot":
+      // Retrieve the URL for the requested game using a simplified variable assignment
+      const gameUrl = gameUrls[params.game];
+      // Send the URL for the requested game
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)